R/checkSummations.R

```diff
@@ -32,17 +33,20 @@ checkSummations <- function(mifFile, outputDirectory = ".", template = "AR6", su
 
   data <- quitte::as.quitte(mifFile, na.rm = TRUE)
 
-  summationGroups <- getSummations(summationsFile)
-
-  if (summationsFile %in% names(summationsNames())) {
-    summationsFile <- gsub(".*piamInterfaces", "piamInterfaces", summationsNames(summationsFile))
-  }
+  if (summationsFile == "extractVariableGroups") {
```

Comment:
to avoid that this fails with `summationsFile = NULL`:
```suggestion
  if (isTRUE(summationsFile == "extractVariableGroups")) {
```